extensions-contrib/kubernetes-overlord-extensions/src/test/resources/expectedNoopJob.yaml

```diff
@@ -4,6 +4,10 @@ metadata:
   name: "id"
   labels:
     druid.k8s.peons: "true"
+    druid.io/task.id: "id"
+    druid.io/task.type: "noop"
+    druid.io/task.group.id: "id"
+    druid.io/task.datasource: "datasource"
```

Comment:
can we use the same convention as `druid.k8s.peons`? 
so `druid.io/task.id` becomes `druid.task.id` - the latter is also easier to type.